src/test/perf_local.cc

```diff
@@ -950,6 +1022,10 @@ TestInfo tests[] = {
     "Lfence instruction"},
   {"sfence", sfence,
     "Sfence instruction"},
+  {"dmb", dmb,
```

Comment:
if dmb and dsb do not exist on non-arm architectures,  can we guard these two structures with `#if defined(__arm__) || defined(__aarch64__)` as well?